docs/howtos/layers/shapes.md

```diff
@@ -290,125 +528,51 @@ nbscreenshot(viewer, alt_text="Add new ellipses to an existing shapes layer usin
 viewer.close()
 ```
 
-## Creating a new shapes layer
+### Non-editable mode
 
-As you can add new shapes to a shapes layer using the various shape creation
-tools, it is possible to create a brand-new empty shapes layers by clicking the
-new shapes layer button above the layers list. The shape of the shapes layer is
-defined by the shapes inside it, and so as you add new shapes the shape will
-adjust as needed. The dimension of the new shapes layer will default to the
-largest dimension of any layer currently in the viewer, or to 2 if no other
-layers are present in the viewer.
+To disable editing of the shapes layer, set the `editable` property of the layer
+to `False`.
 
-## Non-editable mode
+As noted in [3D rendering](#3d-rendering), below, the shapes layer is not
+editable when using 3D rendering.
 
-If you want to disable editing of the shapes layer you can set the `editable`
-property of the layer to `False`.
+### 3D rendering
 
-As note in the section on 3D rendering, when using 3D rendering the shapes layer
-is not editable.
+All layers can be rendered in both 2D and 3D.
 
-## 3D rendering of shapes
-
-All our layers can be rendered in both 2D and 3D mode, and one of our viewer
-buttons can toggle between each mode. The number of dimensions sliders will be 2
-or 3 less than the total number of dimensions of the layer. See for example the
-[`examples/nD_shapes.py`](https://github.com/napari/napari/blob/main/examples/nD_shapes.py)
-to see shapes in both 2D and 3D:
+The number of dimensions sliders will be 2 or 3 less than the total number of
+dimensions of the layer, allowing you to browse volumetric timeseries data and
+other high dimensional data. See [](../../gallery/nD_shapes) to see shapes in
+both 2D and 3D:
 
 ![image: nD shapes](../../images/nD_shapes.webm)
```

Comment:
This is an older version of napari, but not a big deal in my opinion it's just showing rendering.

Reply:
Yeah, I thought this could be refreshed some other time but now that you mention it I'll update too as long as I'm here.